payplug.php

```diff
@@ -24,43 +24,47 @@
 *  International Registered Trademark & Property of PayPlug SAS
 */
 
-if (version_compare(_PS_VERSION_, '1.4', '<'))
-{
-	echo 'Sorry Payplug is not compatible with Prestashop for versions < 1.4. Please delete the payplug directory in the Prestashop modules directory for your Prestashop system to get back to normal.';
+if (!defined('_PS_VERSION_'))
 	exit;
-}
 
 require_once(_PS_MODULE_DIR_.'/payplug/installPayplug.php');
 
-if (!defined('_PS_VERSION_'))
-	exit;
-
 class Payplug extends PaymentModule
 {
 
-	public static $is_active = 1;
-	const PAYMENT_STATUS_PAID = 0;
+	public static $is_active    = 1;
+	const PAYMENT_STATUS_PAID   = 0;
 	const PAYMENT_STATUS_REFUND = 4;
 	const PAYMENT_STATUS_CANCEL = 2;
-	const URL_AUTOCONFIG = 'https://www.payplug.fr/portal/ecommerce/autoconfig';
+	const URL_AUTOCONFIG        = 'https://www.payplug.fr/portal/ecommerce/autoconfig';
+	/** Url to sandbox */
+	const URL_TEST_AUTOCONFIG   = 'https://www.payplug.fr/portal/test/ecommerce/autoconfig';
 
 	public function __construct()
 	{
 		$this->name = 'payplug';
 		$this->tab = 'payments_gateways';
-		$this->version = '0.9.6';
+		// Update version
+		$this->version = '0.9.7';
 		$this->author = 'PayPlug';
 		$this->module_key = '1ee28a8fb5e555e274bd8c2e1c45e31a';
+
 		parent::__construct();
 
 		// Backward compatibility
-		require(_PS_MODULE_DIR_.$this->name.'/backward_compatibility/backward.php');
+		if (version_compare(_PS_VERSION_, '1.4', '>'))
```

Comment:
You need only the backward compatibility on PS 1.4, so a better condition would be ```if (version_compare(_PS_VERSION_, '1.5', '<'))```